reference/docs-conceptual/community/contributing/powershell-style-guide.md

```diff
@@ -0,0 +1,492 @@
+
+  For example:
+
+  ~~~markdown
+  The following code uses `Get-ChildItem` to list the contents of `C:\Windows` and assigns
+  the output to the `$files` variable.
+
+  ```powershell
+  $files = Get-ChildItem C:\Windows
+  ```
+  ~~~
+
+- When referring to a parameter by name, the name should be **bold**. When illustrating the use of
+  a parameter with the hyphen prefix, the parameter should be wrapped in backticks. For example:
+
+  ```markdown
+  The parameter's name is **Name**, but it is typed as `-Name` when used on the command
+  line as a parameter.
+  ```
+
+- When referring to external commands (EXEs, scripts, etc.), the command name should be bold, all
+  lowercase (or capitalized if at the beginning of a sentence), and include the appropriate file
+  extension. For example:
+
+  ```markdown
+  For example, on Windows systems, you can use the `net start` and `net stop` commands
+  to start and stop a service. **Sc.exe** is another service control tool for Windows.
+  That name does not fit into the naming pattern for the **net.exe** service commands.
+  ```
+
+- When showing example usage of an external command, the example should be wrapped in backticks.
+  When there is a name collisions with an alias you must include the file extension in the command
+  example. For example:
+
+  ```markdown
+  To start the spooler service on a remote computer named DC01, you type `sc.exe \\DC01 start spooler`.
+  ```
+
+- When writing a conceptual article (as opposed to reference content), the first instance of a
+  cmdlet name should be hyperlinked to the cmdlet documentation. Do not use backticks, bold, or
+  other markup inside the brackets of a hyperlink.
+
+  For example:
+
+  ```markdown
+  This [Write-Host](/powershell/module/Microsoft.PowerShell.Utility/Write-Host) cmdlet
+  uses the **Object** parameter to ...
+  ```
+
+  For more information, see [Hyperlinks](#hyperlinks) section of this article.
+
+### Images
+
+The syntax to include an image is:
+
+```markdown
+![[alt text]](<folderPath>)
+
+Example:
+![Introduction](./media/overview/Introduction.png)
+```
+
+Where `alt text` is a brief description of the image and `<folder path>` is a relative path to the
+image. Alternate text is required for screen readers for the visually impaired. It is also useful in
+case there is a site bug where the image cannot be rendered.
+
+Images should be stored in a `media/<article-name>` folder within the folder containing the article.
+Images should not be shared between articles. Create a folder that matches the filename of your
+article under the `media` folder. Copy the images for that article to that new folder. If an image
+is used by multiple articles, each image folder must have a copy of that image file. This practice
+prevents a change to an image in one article affecting another article.
+
+The following image file types are supported: *.png", *.gif", *.jpeg", *.jpg", *.svg
```

Comment:
These should get code fenced or escaped due to the asterisk.  Also, they mostly end with quotes even though they didn't start with them.

Reply:
fixed